docs/release-notes/8.3.0-1.md

```diff
@@ -0,0 +1,94 @@
+# Percona Server for MySQL 8.3.0-1 (2024-0-0)
+
+Get started with [Quickstart Guide for Percona Server for MySQL].
+
+[Percona Server for MySQL 8.3.0-1] includes all the features and bug fixes available in the
+MySQL 8.3 Community Edition in addition to enterprise-grade features developed by Percona.
+
+## Release highlights
+
+This release merges the MySQL 8.3 code base. Improvements and bug fixes introduced by Oracle for MySQL 8.3 and included in Percona Server for MySQL are the following:
+
+* Implements tagged GTIDs (Global Transaction Identifiers) to group related transactions. This helps enhance replication management and simplify tracking transaction dependencies.
```

Comment:
Also our Binary Log UDFs were adapted to work properly with Tagged GTIDs
https://github.com/percona/percona-server/commit/92cb30d2fcca18ad86f387259af4a2b4e520ea3d
`Make sure Binlog Utils UDFs are compatible with new GTID format (tagged GTIDs)` item from the merge TODO list